README.md

```diff
@@ -85,10 +85,10 @@ embeds.analogy("Berlin", "Deutschland", "Amsterdam")
 ~~~
 
 More usage examples can be found in the
-[examples](https://github.com/danieldk/finalfusion-python/tree/master/examples)
+[examples](https://github.com/finalfusion/finalfusion-python/tree/master/examples)
 directory.
 
 ## Where to go from here
 
-  * [finalfrontier](https://github.com/danieldk/finalfrontier)
-  * [rust2vec](https://github.com/danieldk/rust2vec)
+  * [finalfrontier](https://github.com/finalfusion/finalfrontier)
```

Comment:
Thanks for updating this. Maybe these links should be updated to:

https://finalfusion.github.io/finalfrontier
https://finalfusion.github.io/

? Any opinions?

Reply:
Sure, I think it's nicer to use references to the webpage over github. People will still be able to find the github repos from the webpage.

Reply:
Indeed.